manifests/init.pp

```diff
@@ -1,85 +1,32 @@
-# This Class sets up the docker environment and image(s)
+# Set up docker images
 
 class dockeragent (
-  $registry = undef,
-  $yum_server = 'master.puppetlabs.vm',
```

Comment:
These options need to pass through to the docker::image type so that the usage of the dockeragent class doesn't change.